np2.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def np_random_vector():
+    """
+    Function that creates a vector of length 10 with values
+    evenly distributed between 5 and 50
+    :return: Vector
+    """
+    vector = np.random.randint(5, 51, 10)
+    return vector
+
+
+if __name__ == '__main__':
+    print(np_random_vector())
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add